packages/client/src/components/block-defaults/stepper-block/config.tsx

```diff
@@ -36,11 +28,21 @@ export const config: BlockConfig<StepperBlockDef> = {
                 {
                     description: 'Steps',
                     render: ({ id }) => (
-                        <SwitchSettings
+                        <StepperStepsSettings
+                            id={id}
+                            label="Steps"
+                            path="steps"
+                        />
+                    ),
+                },
+                {
+                    description: 'Active Step',
+                    render: ({ id }) => (
+                        <InputSettings
                             id={id}
-                            label="Enable Multi Select"
-                            path="multiple"
-                            description="This setting will enable the multi-select feature on the select input"
+                            label="Active Step"
+                            path="active"
+                            type="number"
```

Comment:
![image](https://github.com/user-attachments/assets/3a7e5175-20d7-492f-a95c-c99e8b9600b3)
![image](https://github.com/user-attachments/assets/f157e1fd-e001-41b4-8247-fc83f3e3ec4e)

I need to be able to reference a variable from the notebook to determine what step is active.  The way this is setup at the moment, does not allow any dynamic behavior.

Meaning a button click outside of the stepper cannot determine the nextStep.

The StepperBlock, should parseInt() if we are worried about the setting it to the right data type

For Example: 

![image](https://github.com/user-attachments/assets/e90af2f3-136e-46f4-9b0e-dcd21d294a13)